ocw/lib/decorators.py

```diff
@@ -0,0 +1,20 @@
+def filterAfterTime(*args, **kwargs):
+    def wrapper(func):
+        from datetime import datetime, timedelta
+        time_limit = datetime.now() - timedelta(hours=kwargs['hours'])
+
+        def time_filtered_list(*args, **kwargs):
+            dat = func(*args)
+            dt_frm = datetime.fromisoformat
+            return [k['name'] for k in dat if dt_frm(
+                    k['validAfterTime'][:-1]) < time_limit]
```

Comment:
I assume `validAfterTime` is field from Google service account field ?

Reply:
yeah for such common function, I think a you would need to introduce a common class object or just do it in `gce.py`

Reply:
*changes applied and waiting for feedback*